validator/client/runner.go

```diff
@@ -96,6 +98,11 @@ func run(ctx context.Context, v Validator) {
 				continue
 			}
 
+			if err := v.UpdateProtections(ctx, slot); err != nil {
+				log.WithError(err).Error("Could not update validator protection")
+				// Should we `continue` here or no?
```

Comment:
If the history can't be updated, should it `continue` or allow the validator to do their roles for the slot?